Client/Frontend/Browser/Authenticator.swift

```diff
@@ -57,7 +57,7 @@ class Authenticator {
 
             let logins = cursor.compactMap {
                 // HTTP Auth must have nil formSubmitUrl and a non-nil httpRealm.
-                return $0?.formSubmitUrl == nil && $0?.httpRealm != nil ? $0 : nil
+                return $0?.fields.formActionOrigin == nil && $0?.fields.httpRealm != nil ? $0 : nil
```

Comment:
I plan to create computed fields in the `RustLogins` file so that I can minimize the need for property changes like this.